pkgs/tools/misc/mongodb-tools/assertions.nix

```diff
@@ -0,0 +1,19 @@
+{ goPackages, fetchurl, callPackage }:
```

Comment:
You're also adding two versions of this. Is that intentional? If so, I'd note the duplication in a comment along with some criterion for when the duplication will no longer be necessary (if applicable)